packages/main/src/SideNavigation.hbs

```diff
@@ -0,0 +1,30 @@
+<div class="ui5-sn-root">
+    {{#if items.length}}
+        <div
+            class="ui5-sn-content"
+            @ui5-_click="{{handleItemClick}}"
+            @ui5-_forward-focus="{{_focusOtherList}}"
+            @ui5-_focus_next="{{focusNext}}"
+            @ui5-_focus_previous="{{focusPrevious}}"
+        >
+            <div class="ui5-sn-flex-wrapper">
+                <slot></slot>
+            </div>
+            <span class="ui5-sn-middle-focus-helper" tabindex="-1"></span>
+        </div>
+    {{/if}}
+
+    {{#if fixedItems.length}}
+        <div class="ui5-sn-bottom-content">
+            <div
+                class="ui5-sn-flex-wrapper"
+                @ui5-_click="{{handleItemClick}}"
```

Comment:
would not it work with standard @ click, instead of firing ui5-_click on mouseup